pkg/shp/cmd/build/upload.go

```diff
@@ -203,16 +203,21 @@ func (u *UploadCommand) performDataStreaming(target *streamer.Target) error {
 		return nil
 	}
 
-	log.Printf("Streaming '%s' to the Build POD '%s'...", u.sourceDir, target.Pod)
+	fmt.Fprintf(os.Stdout, "Streaming %q to the Build POD %q ...\n", u.sourceDir, target.Pod)
```

Comment:
Any particular reason to switch from `log` to `fmt`?  The rest of the code seems to favor `log` still. We create a bit of an inconsistency in the output:
```
2022/10/17 11:36:59 Creating a BuildRun for 'default/local' Build...
2022/10/17 11:36:59 BuildRun 'local-4nctc' created!
Streaming "/Users/mdiester/git/sample-go" to the Build POD "local-4nctc-kr7fw-pod" ...
Uploading local source...  100% [===============] (43 MB/s)
2022/10/17 09:37:15 Removing lock-file at '/tmp/waiter.lock' (23 PID)
```

Personally, I have no strong preference here to be honest. Honestly, I think I also contributed to the inconsistencies by using `fmt` in the bundle code.

Reply:
I have been following your bundle code for the output.